qldpc/objects.py

```diff
@@ -227,9 +231,9 @@ def __init__(
     def get_min_rank(
         cls,
         group: abstract.Group,
-        subset_a: Collection[abstract.GroupMember],
-        subset_b: Collection[abstract.GroupMember],
-    ) -> Literal[0, 1, 2]:
+        subset_a: "Collection[abstract.GroupMember]",
+        subset_b: "Collection[abstract.GroupMember]",
+    ) -> "Literal[0, 1, 2]":
```

Comment:
(assuming prior)
```suggestion
        subset_a: Collection[abstract.GroupMember],
        subset_b: Collection[abstract.GroupMember],
    ) -> Literal[0, 1, 2]:
```
(ditto below)